packages/office-addin-test-helpers/test/test.ts

```diff
@@ -43,5 +49,5 @@ async function _sendTestData(): Promise<boolean> {
     testData[valueKey] = testValue;
     testValues.push(testData);
 
-    return testHelper.sendTestResults(testValues, port);
+    return testHelper.sendTestResults(testValues, defaultHttpsPort);
```

Comment:
shoudn't this just have `useHttps: boolean = true` in the sendTestResults method? Then you don't need this extra parameter when calling it here. Is there any reason to sendTestResults() using HTTP and not HTTPS?

Reply:
I see this is a port, but it would be better to have `useHttps: boolean` parameter here instead of the port. You need to change both the procotol and matching port number, right?

Reply:
the only problem is that this will be a breaking change that will require udpate to all the tests in all the project repos.  Come to think of it, the suggested change to pingTestServer to take a https:boolean parameter is also a breaking change. I will like to avoid this if possible

Reply:
I am going to hold off on changing this so I don't have to update all the projects to use the boolean useHttps instead of the port